nolooking/docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
       PROXY_AUTH_WHITELIST: "/pj"
   
   server:
-    image: chaincase/nolooking:v0.2.2-alpha@sha256:e4a95443a05b44659f55a20c592142d59a4d8f54830a71f4c3d22a3a6bf88801
+    image: chaincase/nolooking:v0.3.0-alpha@sha256:733025b09682e219d06d14b27d487b8af4ae78590e625867460be8229eec8d3e
```

Comment:
It looks like the Docker image digest is wrong. Here is the digest for `amd64` only. To get the multi-arch digest, you can run `docker pull <image>:<tag>` and it will display the digest at the bottom once complete.

In this case it should be: `sha256:ba879c089e6609cb85869081c18bc36086984276c5cd73bce2a24ff5f53cc1f0`

Reply:
💯